src/pyvesync/vesyncfan.py

```diff
@@ -1353,9 +1399,10 @@ def set_night_light_brightness(self, brightness: int) -> bool:
 
     def set_humidity_mode(self, mode: str) -> bool:
         """Set humidifier mode - sleep or auto."""
-        if mode.lower() not in ['sleep', 'auto']:
-            logger.debug('Invalid humidity mode used (sleep or auto)- %s',
-                         mode)
+        if mode.lower() not in ['sleep', 'auto', 'humidity']:
+            logger.warning('Invalid humidity mode used (sleep, auto, or '
+                           'humidity)- %s',
+                           mode)
```

Comment:
Is there a difference between auto and humidity? Are there three modes on all devices? I only see auto and sleep mode on the other humidifiers. May have to put feature logic to adjust mode settings per device.

Reply:
I haven't actually tried `auto`. I based this PR off of what I was see in my captures. Agreed on adding the feature logic. 

TBH, with all of the devices I am seeing in their app, this pattern is going to be outgrown pretty quick. :)

```JSON
{
  "traceId": "[TRACE_ID]",
  "payload": {
    "method": "setHumidityMode",
    "data": {
      "mode": "humidity"
    },
    "source": "APP"
  },
  "token": "[TOKEN]",
  "appVersion": "VeSync 3.2.6 build6",
  "configModule": "WFON_AHM_LUH-A602S-WUS_US",
  "method": "bypassV2",
  "cid": "[CID]",
  "debugMode": false,
  "deviceRegion": "US",
  "phoneBrand": "iPhone 12 Pro",
  "userCountryCode": "US",
  "acceptLanguage": "en",
  "accountID": "[ACCOUNTID]",
  "timeZone": "America\\/Chicago",
  "phoneOS": "iOS 15.4"
}
```

Reply:
`auto` didn't work for me. I changed my PR to simplify the interface. The caller won't have to know the distinction with this approach.

Reply:
yeah, I can confirm with my testing that `auto` seems to have been renamed to `humidity` for the LV600S. 

looking forward to seeing this PR soon.